pages/executive/[proposal-id].tsx

```diff
@@ -254,7 +254,7 @@ const ProposalView = ({ proposal }: Props): JSX.Element => {
                 tabListStyles={{ pl: [3, 4] }}
                 tabTitles={['Spell Details']}
                 tabPanels={[
-                  <div key={1} sx={{ p: [3, 4] }}>
+                  <div key={'spell'} sx={{ p: [3, 4] }}>
```

Comment:
Should these `key` changes be included?

Reply:
they don't hurt, i almost prefer them to the indexed key that can fail when moving components

Reply:
Just wanted to make sure if it was intended for this PR but I agree 👍